pages/linux/aur.md

```diff
@@ -19,3 +19,7 @@
 - [u]pgrade local repository packages:
 
 `aur sync --upgrades`
+
+- Install a package without viewing changes in vim:
```

Comment:
```suggestion
- Install a package without viewing changes in Vim:
```

Reply:
Can you please check whether the editor launched (without `--noview`) respects the `EDITOR` and/or `VISUAL` environment variables? If so I think this description should reflect that.

Reply:
For now, I have not found any code that indicates that. However, `aur-view` uses the value of the environment variable `AUR_PAGER`.